src/Window.hpp

```diff
@@ -348,6 +348,7 @@ class CWindow {
     void                     addWindowDeco(std::unique_ptr<IHyprWindowDecoration> deco);
     void                     updateWindowDecos();
     void                     removeWindowDeco(IHyprWindowDecoration* deco);
+    bool                     checkInputOnDecos(const eInputType, const Vector2D&, std::any = 0);
```

Comment:
leave any uninitialized, 0 might give it an `int` type with value `0` instead of it being empty

Reply:
Currently there is no `data` set with `INPUT_TYPE_DRAG_START`, so this is how I called it in `KeybindManager`: `pWindow->checkInputOnDecos(INPUT_TYPE_DRAG_START, mouseCoords);`

If I remove the default 0, then this won't work (won't compile).
Or should I always pass a third parameter? If yes, what type?

Reply:
`= {}`

Reply:
Done, rebased.